patches/Codeigniter_Framework/v3.1.13/10-php82_support.patch

```diff
@@ -0,0 +1,123 @@
+From fb1256a5b009b6264fbc85be44e0d97654d3fcd9 Mon Sep 17 00:00:00 2001
+From: George Petculescu <gxgpet@gmail.com>
+Date: Sun, 6 Nov 2022 16:13:43 +0200
+Subject: [PATCH] Adding PHP 8.2 support
+
+---
+ .github/workflows/test-phpunit.yml          | 20 +++++++++++++++++++-
+ system/core/Loader.php                      |  1 +
```

Comment:
Instead of having patches here, can we refer to pr's on the upstream?

Reply:
Then maybe we should raise that to upstream?

Reply:
Hmm, okay for now, we should probably also upgrade to ci4 then given no support for ci3 anymore?